invenio_circulation/search/api.py

```diff
@@ -42,7 +44,10 @@ def search_by_pid(item_pid=None, document_pid=None, filter_states=None,
     elif item_pid:
         search = search.filter("term", item_pid=item_pid)
     else:
-        raise ValueError("Must specify item_pid or document_pid param")
+        raise PropertyRequired(description=(
```

Comment:
I think it should be named `RequiredParameterMissingError` not property. Property for me is something that an object owns, and we pass those values as search parameters.